### DISCUSSION TITLE
Allow Drag & Drop for Bookmark Layouts

### DISCUSSION THREAD

**Discussion by pdesuze** [zombieFox/nightTab#345] [Ideas]
This would be nice to have for housekeeping purposes.

**Comment by metruzanca** [comment]
Do you mean this?

https://user-images.githubusercontent.com/32832892/155343492-f7c88920-cc02-4dde-9888-19fe5025a08e.mp4

If not, could you give us a bit more information about what you mean by "bookmark layouts"?

**Comment by pdesuze** [comment]
Can you do it with individual bookmarks?On 23 Feb 2022 10:51, Samuele Zanca ***@***.***> wrote:
Do you mean this?
https://user-images.githubusercontent.com/32832892/155343492-f7c88920-cc02-4dde-9888-19fe5025a08e.mp4

**Reply by metruzanca** [reply]
Yeah, that same icon used for dragging bookmark groups is also below each bookmark (in the center). You can both drag bookmarks inside the group to change the order and between groups.
